src/panels/lovelace/entity-rows/hui-sensor-entity-row.ts

```diff
@@ -7,12 +7,16 @@ import "./hui-error-entity-row";
 
 import { HomeAssistant } from "../../../types";
 import { EntityRow, EntityConfig } from "./types";
+import computeStateDisplay from "../../../common/entity/compute_state_display";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
 
 interface SensorEntityConfig extends EntityConfig {
   format?: "relative" | "date" | "time" | "datetime";
+  display?: string;
```

Comment:
We should specify which values we can expect, like `format`.

Reply:
Then we would need to limit which attributes can be displayed.